docker-build.sh

```diff
@@ -12,9 +12,17 @@ fi
 
 if [ "$#" -eq 1 ] && [ "$1" = "bash" ]; then
     # run interactive shell - using ROOT user
-    exec docker run -it  --rm -v "$(pwd):/src" -w /src -e TARGET -e SYSPAGE -e CONSOLE --entrypoint bash $DOCKER_IMG_NAME
+    exec docker run -it --rm -v "$(pwd):/src" -w /src -e TARGET -e SYSPAGE -e CONSOLE --entrypoint bash $DOCKER_IMG_NAME
 else
-    # run build - use our own UID/GID to create files with correct owner
-    exec docker run -it --user "$DOCKER_USER" --rm -v "$(pwd):/src:delegated" -w /src "${TMPFS_OVERLAY[@]}" -e TARGET -e SYSPAGE -e CONSOLE $DOCKER_IMG_NAME "$@"
+    # FIXME: run build - use our own UID/GID to create files with correct owner
+    #exec docker run -it --user "$DOCKER_USER" --rm -v "$(pwd):/src:delegated" -w /src "${TMPFS_OVERLAY[@]}" -e TARGET -e SYSPAGE -e CONSOLE $DOCKER_IMG_NAME "$@"
+
+    # FOR NOW: run build as root to be able to overwrite files installed in toolchain
+    docker run -it --rm -v "$(pwd):/src:delegated" -w /src "${TMPFS_OVERLAY[@]}" -e TARGET -e SYSPAGE -e CONSOLE $DOCKER_IMG_NAME "$@"
+
+    # FIX file ownership in "_build"
+    if [ ${#TMPFS_OVERLAY[@]} -eq 0 ]; then # only if not using ramdisk
+        docker run -it --rm -v "$(pwd):/src:delegated" -w /src --entrypoint bash phoenixrtos/build -c "chown -R $DOCKER_USER _build/$TARGET/"
+    fi
```

Comment:
It shouldn't also fix `_boot` permissions?

Reply:
Yes, you're right, probably `_fs/$TARGET` permissions also. Will fix in the moment